### PR TITLE
Updating commands.json

### PR DESCRIPTION
**Pull request by chayim**
The modules commands, until now did not indicate whether or not they were readonly. This is necessary as it currently impacts a variety of projects.

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -131,6 +131,9 @@
                         "token": "V",
                         "optional": true
                     }
+                ],
+                "command_flags": [
+                    "readonly"
```
Comment (MeirShpilraien) [inline commands.json:136]: We need to validate that RedisInsight can handle this

Comment (thomascaudron) [reply commands.json:136]: Hey @ViktarStarastsenka, can you confirm that this is supported by RedisInsight?

Comment (ViktarStarastsenka) [reply commands.json:136]: it is not supported by now
these command flags will be ignored for now, but I'll raise a ticket to handle them

Comment (ViktarStarastsenka) [reply commands.json:136]: please postpone merging this PR - we'll test it in RedisInsight first

Comment (ViktarStarastsenka) [reply commands.json:136]: tested, you can safely merge the PR

**Review by thomascaudron — APPROVED** [review]

**Pull request merged**